common/src/main/scala/org/apache/celeborn/common/meta/WorkerInfo.scala

```diff
@@ -232,8 +232,12 @@ class WorkerInfo(
   }
 
   override def hashCode(): Int = {
-    val state = Seq(host, rpcPort, pushPort, fetchPort, replicatePort)
-    state.map(_.hashCode()).foldLeft(0)((a, b) => 31 * a + b)
+    var result = host.hashCode()
+    result = 31 * result + rpcPort.hashCode()
+    result = 31 * result + pushPort.hashCode()
+    result = 31 * result + fetchPort.hashCode()
+    result = 31 * result + replicatePort.hashCode()
+    result
```

Comment:
Can we cache the hashCode value? seems this method called many times

Reply:
The calculation here is relatively cheap, and the cache will not be much improved, so we may make it simpler.